app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -6,6 +6,10 @@ def call(toolbar_name)
     build_toolbar(toolbar_name)
   end
 
+  def call_by_class(toolbar_class)
+    build(toolbar_class)
+  end
```

Comment:
@eclarizio : you are giving a class to `build` and you are giving a class to `call_by_class` ;-)

So let me ask again: what is the difference? Why do we have 2 methods?